libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -69,20 +69,20 @@
     <parameter name="layer" type="string" value="" uiname="Layer"/>
     <parameter name="default" type="float" value="0.0" uiname="Default Color"/>
     <input name="texcoord" type="vector2" defaultgeomprop="UV0" uiname="Texture Coordinates"/>
-    <parameter name="uaddressmode" type="string" value="periodic" enum="black,clamp,periodic" uiname="Address Mode U"/>
-    <parameter name="vaddressmode" type="string" value="periodic" enum="black,clamp,periodic" uiname="Address Mode V"/>
+    <parameter name="uaddressmode" type="string" value="periodic" enum="black,clamp,periodic,mirror,default" uiname="Address Mode U"/>
+    <parameter name="vaddressmode" type="string" value="periodic" enum="black,clamp,periodic,mirror,default" uiname="Address Mode V"/>
     <parameter name="filtertype" type="string" value="linear" enum="closest,linear,cubic" uiname="Filter Type"/>
     <parameter name="framerange" type="string" value="" uiname="Frame Range"/>
     <parameter name="frameoffset" type="integer" value="0" uiname="Frame Offset"/>
-    <parameter name="frameendaction" type="string" value="black" enum="black,clamp,periodic" uiname="Frame End Action"/>
+    <parameter name="frameendaction" type="string" value="black" enum="black,clamp,periodic,default" uiname="Frame End Action"/>
```

Comment:
Was this intentional? If so I guess all versions should have it?

Reply:
Good catch. String replacement error. I'll fix this up.